src/extensions/replay/sessionrecording.ts

```diff
@@ -586,12 +585,8 @@ export class SessionRecording {
         if (isUserInteraction) {
             this._lastActivityTimestamp = event.timestamp
             if (this.isIdle) {
-                // Remove the idle state if set and trigger a full snapshot as we will have ignored previous mutations
+                // Remove the idle state
                 this.isIdle = false
-                this._tryAddCustomEvent('sessionNoLongerIdle', {
```

Comment:
What's the reason behind removing events like this? I'd have imagined they were somewhat useful for users in the inspector